src/fu-tool.c

```diff
@@ -660,6 +666,46 @@ fu_util_install_blob (FuUtilPrivate *priv, gchar **values, GError **error)
 	return fu_util_prompt_complete (priv->completion_flags, TRUE, error);
 }
 
+static gboolean
+fu_util_verify_blob (FuUtilPrivate *priv, gchar **values, GError **error)
+{
+	g_autoptr(FuDevice) device = NULL;
+	g_autoptr(GBytes) blob_fw = NULL;
+
+	/* invalid args */
+	if (g_strv_length (values) == 0) {
+		g_set_error_literal (error,
+				     FWUPD_ERROR,
+				     FWUPD_ERROR_INVALID_ARGS,
+				     "Invalid arguments");
+		return FALSE;
+	}
+
+	/* load engine */
+	if (!fu_util_start_engine (priv, FU_ENGINE_LOAD_FLAG_NONE, error))
+		return FALSE;
+
```

Comment:
I think it would make sense to try to touch the filename here first to avoid failing at the end of a potentially lengthy operation.

Reply:
You mean write a zero-length file? Also, what should we do if the file already exists?

Reply:
Yeah I was thinking write zero length file. If it already exists show a message it's been overwritten?